candid/xtc-history-e2e.did

```diff
@@ -0,0 +1,46 @@
+type TransactionId = nat64;
+
+type EventDetail = variant {
+    Transfer : record {
+        from : principal;
+        to   : principal;
+    };
+    // DEX events
+    Mint     : record {
+        to   : principal;
+    };
+    Burn     : record {
+        from : principal;
+        to   : principal;
+    };
+    CanisterCalled : record {
+        canister : principal;
+        method_name: text;
+    };
+    CanisterCreated : record {
+        canister : principal;
```

Comment:
This will need the from address

Reply:
That change should not be in the scope of this PR. It should be handled in another PR, either after merging this one or before.